csfieldguide/templates/interactives/sorting-algorithm-comparison.html

```diff
@@ -19,7 +19,7 @@
   <img id="sorting-algorithm-comparison-selection-sort-animation" class="sorting-algorithm-comparison-preload" src="{% static 'interactives/sorting-algorithm-comparison/img/selection-sort.gif' %}">
   <img id="sorting-algorithm-comparison-quick-sort-animation" class="sorting-algorithm-comparison-preload" src="{% static 'interactives/sorting-algorithm-comparison/img/quick-sort.gif' %}">
   
-  <link rel="stylesheet" href="{% static 'interactives/sorting-algorithm-comparison/css/sorting-algorithm-comparison.css' %}">
-  <script src="{% static 'interactives/sorting-algorithm-comparison/js/sorting-algorithm-comparison.js' %}"></script>
+  <link rel="stylesheet" href="{% static 'interactives/sorting-algorithm-comparison/css/style.css' %}">
```

Comment:
Why are these files renamed?

Reply:
consistency, there isn't really a standard amongst out interactive filenames atm.
Makes just as much sense to call everything `<interactive-slug>.html`/`css`/`js` etc, just `style` and `script` is a convention I've seen a lot. Up for discussion.